.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Cross-Platform CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 3
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [20.18.0]
```

```suggestion
        node: [22, 20, 18]
```

Syntax from https://github.com/actions/setup-node?tab=readme-ov-file#matrix-testing.

---

This is the version we have in the README. We can update the README and add all the specified versions.

---

Sure, it can also be part of this PR.